.github/workflows/test.yml

```diff
@@ -0,0 +1,55 @@
+name: Test
+
+env:
+  OTP_VERSION: 27.3.4.2
+  ELIXIR_VERSION: 1.18.4
+  MIX_ENV: test
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  test:
+    name: mix test (${{ matrix.os }})
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-24.04
+          - macos-15
+          - windows-2022
```

Comment:
Just to confirm: What do you think about using `ubuntu-latest` and `macos-latest` here? The reason why I think so is that `code-analysis` uses `ubuntu-latest`, and we've also decided to use `-latest` in `nif-precompile` in #107.
If that's the case, we'd prefer to use `windows-latest`, but I encountered that `mix test` doesn't work well with it,,,
https://github.com/biyooon-ex/zenohex/pull/116#issuecomment-3263457899

Reply:
I don't have any opinion about this. I also encountered the issue you mentioned, so I just specified the exact each OS versions.

It might be good to use `latest` for ubuntu and macos, and give a note to windows `WHY`. If you like the way, I will change the file.